doc/source/examples/cfd-dem/square-fluidized-bed/square-fluidized-bed.rst

```diff
@@ -0,0 +1,389 @@
+           set id		   = 5
+           set type              = slip
+        end
+       subsection bc 4
+           set id		   = 2
+           set type              = function
+           subsection u
+            set Function expression = 0
+           end
+           subsection v
+            set Function expression = 2
+           end
+           subsection w
+            set Function expression = 0
+           end
+       end
+    end
+
+The additional sections for the CFD-DEM simulations are the void fraction subsection and the CFD-DEM subsection. These subsections are described in detail in the `CFD-DEM parameters <../../../parameters/unresolved_cfd-dem/unresolved_cfd-dem.html>`_ .
+
+Void fraction
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Since we are calculating the void fraction using the packed bed of the DEM simulation, we set the mode to "dem". For this, we need to read the dem files which we already wrote using check-pointing. We therefore set the read dem to "true" and specify the prefix of the dem files to be read. In order to ensure that our void fraction projection is bounded, we choose an upper bound limit of 1. We decide not to lower bound the void fraction and thus attributed a value of 0 to the L2 lower bound parameter. We now choose a smoothing factor for the void fraction as to reduce discontinuity which can lead to oscillations in the velocity. The factor we choose is around the square of twice the particle's diameter. 
+ 
+.. code-block:: text
+
+    subsection void fraction
+        set mode = dem
+        set read dem = true
+        set dem file name = dem
+        set l2 smoothing factor = 0.000005
+        set l2 lower bound = 0
+        set l2 upper bound = 1
+        set bound void fraction = true
+    end
+
+CFD-DEM
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We also enable grad_div stabilisation in order to improve local mass conservation. The void fraction time derivative is enabled to account for the time variation of the void fraction. For certain simulations, this parameter should be disabled to improve stability of the solver.
+
+.. code-block:: text
+
+    subsection cfd-dem
+        set grad div = true
+        set void fraction time derivative = true
+        set drag force = true
+        set buoyancy force = true
+        set shear force = false
+        set pressure force = false
+        set drag model = difelice
+        set post processing = true
+   	set inlet boundary id = 2
+   	set outlet boundary id = 3
+   	set coupling frequency = 100
+        set vans model = modelB
+    end
+    
+We determine the drag model to be used for the calculation of particle-fluid forces. Currently, Difelice, Rong and Dallavalle models are supported. Other optional forces that can be enabled are the buoyancy force, the shear force and the pressure force. We only decide to enable drag and buoyancy as for air, the other forces are considered to be negligible. The VANS model we are solving is model B. Other possible option is model A.
```

Comment:
Should we add a reference for the models?

Reply:
They are referenced in the explanation of the cfd-dem parameter file